NETPROGRAM/BasicExercise/Exercise03.cs

```diff
@@ -0,0 +1,60 @@
+﻿
+using System;
+
+namespace NETPROGRAM.BasicExercise
+{
+    public class Exercise03
+    {
+
+        public bool isPrime(int n)
+        {
+
+            if(n < 2)
+            {
+                return false;
+            }
+
+            for(int i=2; i < n / 2; i++)
+            {
+                if (n % i == 0)
+                    return false;
+            }
+
+            return true;
+        }
+
+        public String showMultiPrime(int n)
+        {
+
+            String stringPrime = null;
+            var i = 2;
+
+            if(n <= i)
+            {
+                return n.ToString();
+            }
+
+            do
+            {
+                if (n % i == 0 && isPrime(i))
```

Comment:
Thật ra không cần kiểm tra i là số nguyên tố

```suggestion
                if (n % i == 0)
```

Reply:
done